src/StatusQ/Components/StatusChatToolBar.qml

```diff
@@ -83,6 +83,7 @@ Item {
 
             StatusFlatRoundButton {
                 id: menuButton
+                objectName: "channelMoreOptionsButton"
```

Comment:
Wonder if it should be rather called `chatToolbarMoreOptionsButton` because I could imagine this to conflict with the more options button of channels otherwise (in case those will be tested as well and need a name)

Otherwise I trust your choice

Reply:
Good idea. Updated